gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -141,96 +155,117 @@ public Builder toBuilder() {
     /**
      * The TotalTimeout parameter has ultimate control over how long the logic should keep trying
      * the remote call until it gives up completely. The higher the total timeout, the more retries
-     * can be attempted.
+     * can be attempted. The default value is {@code Duration.ZERO}.
      */
     public abstract Builder setTotalTimeout(Duration totalTimeout);
 
     /**
      * The InitialRetryDelay parameter controls the delay before the first retry. Subsequent retries
-     * will use this value adjusted according to the RetryDelayMultiplier.
+     * will use this value adjusted according to the RetryDelayMultiplier. The default value is
+     * {@code Duration.ZERO}.
      */
     public abstract Builder setInitialRetryDelay(Duration initialDelay);
 
     /**
      * The RetryDelayMultiplier controls the change in retry delay. The retry delay of the previous
      * call is multiplied by the RetryDelayMultiplier to calculate the retry delay for the next
-     * call.
+     * call. The default value is {@code 1.0}.
      */
     public abstract Builder setRetryDelayMultiplier(double multiplier);
 
     /**
      * The MaxRetryDelay puts a limit on the value of the retry delay, so that the
-     * RetryDelayMultiplier can't increase the retry delay higher than this amount.
+     * RetryDelayMultiplier can't increase the retry delay higher than this amount. The default
+     * value is {@code Duration.ZERO}.
      */
     public abstract Builder setMaxRetryDelay(Duration maxDelay);
 
     /**
-     * MaxAttempts defines the maximum number of attempts to perform. If number of attempts reaches
-     * this limit the logic will give up retrying even if the total retry time is still lower than
-     * TotalTimeout.
+     * The MaxAttempts defines the maximum number of attempts to perform. If number of attempts
```

Comment:
For grammar, it's only valid to use "MaxAttempts defines ..." or "The MaxAttempts parameter defines ...". I didn't notice before that there are several comments in the file that use the invalid form like "The MaxAttempts defines ..." Could you fix them? (I prefer the one with "parameter")